pyproject.toml

```diff
@@ -29,6 +29,7 @@ pytest = "5.4.3"
 pytest-cov = "^2.10.1"
 lxml = "^4.6.2"
 pyflakes = "^2.2.0"
+icecream = "^2.1.3"
```

Comment:
Are you planning to leave this in or delete it before this is ready for merge?

Reply:
it's in there just to debug the tests more easily. I don't think there's a need for this in the final PR, however perhaps there's no harm in leaving it in? lmk

Reply:
I would prefer it to be gone, even though it's just a dev dependency. It's a tiny package that just feels pointless and dumb to me, a bit like all the tiny packages in the npm/javascript world.

Reply:
that's valid. I just needed the ease of Rust's `dbg!` to figure a few things out

Reply:
i use `println!` for rust debugging :D

Reply:
That's one way to do it, but `dbg!` can be extra-nice to just plop into existing expressions :)